README.md

```diff
@@ -229,6 +230,14 @@ recorded values will not be recalculated if this value is changed. If you would
 like to change previous values, you would need to write a migration of the data
 to recalculate history.
 
+## Creating accounts
+
+Follow the login screen's "Sign Up" link, and use the registration key specified
+in the server configuration above.
+
+To create an additional account, when logged in, go to the gear menu / Users and
+use the "Add User" button to create a new account without the registration key.
```

Comment:
Could we also add something like:

If you have set up conbench using an SSO provider, but you're looking to make a user name and password for API access, ...

Reply:
Tried again. Better?